pdfbox/src/main/java/org/apache/pdfbox/pdmodel/font/encoding/MacExpertEncoding.java

```diff
@@ -27,6 +27,7 @@ public class MacExpertEncoding extends Encoding
     /**
      * Table of octal character codes and their corresponding names.
      */
+    @SuppressWarnings("OctalInteger")
```

Comment:
I think that's IntelliJ specific and might in turn produce a warning in other IDEs.

Reply:
I won't add that as it is IntelliJ specific